File: Bravi.Minutes/Bravi.Minutes.Web/App/model/minute.js
define(function () {
    var Minute = function (id, date, subject, notes, totalAttendees, attendees) {
        var self = this;

        self.id = ko.observable(id || 0);
        self.date = ko.observable(date || new Date());
        self.subject = ko.observable(subject || '');
        self.notes = ko.observable(notes || '');
        self.totalAttendees = ko.observable(totalAttendees || 0);
        self.attendees = ko.observable(attendees || []);

        return self;
    };

    return Minute;
});

File: Bravi.Minutes/Bravi.Minutes.Core/Entities/Minute.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Bravi.Minutes.Core.Entities
{
    public class Minute
    {
        [Key]
        public int Id { get; set; }
        [Required]
        public DateTime Date { get; set; }
        [Required, MaxLength(100)]
        public string Subject { get; set; }
        [DataType(DataType.Text)]
        public string Notes { get; set; }

        public virtual ICollection<Attendee> Attendees { get; set; } 
    }
}
